custom_components/waste_collection_schedule/waste_collection_schedule/source/moray_gov_uk.py

```diff
@@ -1,32 +1,29 @@
-import logging
-from datetime import datetime
-
 import requests
+
 from bs4 import BeautifulSoup
+from datetime import datetime
 from waste_collection_schedule import Collection  # type: ignore[attr-defined]
 
-_LOGGER = logging.getLogger(__name__)
-
 TITLE = "Moray Council"
 DESCRIPTION = "Source for Moray Council, UK."
 URL = "https://moray.gov.uk"
 TEST_CASES = {
-    "test_str": {"id": "00013734"},
-    "test_int": {"id": 60216},
```

Comment:
I'd like to keep the integer test case. As uses reported issues when providing integers instead of strings to sources. We would like the user to not care about whether they provide a String or an Integer representation of a number

Reply:
Fair point, I'll merge the difference.